### PR TITLE
Complex/more special functions

### PR DESCRIPTION
**Pull request by jtlap**
I need that to write tests for fluxion special functions derivatives

### CODE REVIEW

File: include/eve/module/complex/regular/detail/erf.hpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+#include <eve/module/core.hpp>
+#include <eve/module/math.hpp>
+#include <eve/module/complex.hpp>
+#include <eve/module/complex/regular/traits.hpp>
+#include <iostream>
```
Comment (jfalcou) [inline include/eve/module/complex/regular/detail/erf.hpp:15]: spurrious iostream

File: include/eve/module/complex/regular/eta.hpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+#include <eve/module/complex/regular/detail/special.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
```
Comment (jfalcou) [inline include/eve/module/complex/regular/eta.hpp:13]: spurrious iostream

File: include/eve/module/complex/regular/deta.hpp [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+        -.99999999999975652196,
+        .99999999999998040668,
+        -.99999999999999875788,
+        .99999999999999994183,
+        -.99999999999999999821,
+        .99999999999999999997
+      };
+
+      auto f = zero(as(z));
+      auto n = eve::algo::views::reverse(eve::algo::views::iota_with_step(one(as<real_t>()), k, cm.size()));
```
Comment (DenisYaroshevskiy) [inline include/eve/module/complex/regular/deta.hpp:161]: I'd just write a for loop tbh

File: include/eve/module/complex/regular/detail/erf.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/complex/regular/detail/erf.hpp:15]: done

File: include/eve/module/complex/regular/detail/erfcx.hpp [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+#include <eve/module/core.hpp>
+#include <eve/module/math.hpp>
+#include <eve/module/complex.hpp>
+#include <eve/module/complex/regular/traits.hpp>
+#include <eve/module/complex/regular/faddeeva.hpp>
+#include <iostream>
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/module/complex/regular/detail/erfcx.hpp:16]: done

File: include/eve/module/complex/regular/eta.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/complex/regular/eta.hpp]: dne

File: include/eve/module/complex/regular/deta.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/complex/regular/deta.hpp:161]: why ? Is it faster with a loop ?
My opinion was if we have algos that make the jobs, use them for:
1) they can be improved with time
2) the purpose is clear 
Also when modules will be at hand I thought the aim was "eve include all" anyway, so iota will be here whatsoever,
so it is not an "more includes problem".

Give me some breaking  argument and I'll write a loop

Comment (jtlap) [reply include/eve/module/complex/regular/deta.hpp]: however I have done it deta dose not use algo now

File: include/eve/module/complex/regular/detail/erfcx.hpp (hunk 4 above)
[the remark this answers is not in the document]
Comment (jfalcou) [reply include/eve/module/complex/regular/detail/erfcx.hpp:16]: not there

File: include/eve/module/complex/regular/eta.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/complex/regular/eta.hpp]: all iostream removed

**Pull request merged**